Update the "I'm Locked Out" documentation for using a container CLI

jay-em-Ach opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

1. Added steps to change a lost password through the command line when Home Assistant is running in a container.
2. Added a note that the "Change Password" button does not always show up, but switching to another interface will likely show it (I learned that _after_ figuring out this whole `hass` process).

Replacement for #14477 by @hadim and Home Assistant Community feature request "[Reset password from HA Docker Container](https://community.home-assistant.io/t/reset-password-from-ha-docker-container/215537)."

## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - [x] I made a change to the existing documentation and used the `current` branch.
  - [ ] I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

jay-em-Ach committed:
Update the "I'm Locked Out" documentation for using a container CLI

## Proposed change
Added steps to change a lost password through the command line when Home Assistant is running in a container. Also added a note that the "Change Password" button does not always show up, but switching to another interface will likely show it (I learned that _after_ figuring out this whole `hass` process).

Follow-up to unfinished #14477 by @hadim and https://community.home-assistant.io/t/reset-password-from-ha-docker-container/215537

## Type of change
* [ ]  Spelling, grammar or other readability improvements (`current` branch).
* [x]  Adjusted missing or incorrect information in the current documentation (`current` branch).
* [ ]  Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  
  * [ ]  I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
* [ ]  Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
* [ ]  Removed stale or deprecated documentation.

## Additional information
* Link to parent pull request in the codebase:
* Link to parent pull request in the Brands repository:
* This PR fixes or closes issue:

## Checklist
* [x]  This PR uses the correct branch, based on one of the following:
  
  * [x] I made a change to the existing documentation and used the `current` branch.
  * [ ] I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
* [x]  The documentation follows the Home Assistant documentation [standards](https://developers.home-assistant.io/docs/documenting/standards).

homeassistant commented:
Hi @jay-em-Ach, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18109).

Once you do that we will be able to review and accept this pull request.

Thanks!

frenck commented on source/_docs/locked_out.md:
```diff
@@ -40,7 +42,17 @@ Connect a keyboard and monitor to your device.
+4. `exit` to exit the container command line
+5. `docker restart ha` to restart the container
+
+### Home Assistant Core
```
This is not specific to Home Assistant Core installations

jay-em-Ach replied:
Thanks for the suggestion, @frenck. Corrections are now added.

jay-em-Ach committed:
Update instructions for a forgotten password

- Moved the container command line instructions under the console instructions.
- Used wording from the section introduction to rename the authentication reset instructions.
- Removed mention of using a password manager since the user obviously did not (and HA has not been compatible with some password managers in some browser configurations recently). This is why I did not use one and why I had to figure out how to reset my password, then share the instructions with the community.

jay-em-Ach commented:
Thanks for the suggestion, @frenck. Corrections are now added.

jay-em-Ach committed:
Merge branch 'home-assistant:current' into locked_out-container-instructions

jay-em-Ach committed:
Merge branch 'home-assistant:current' into locked_out-container-instructions

frenck reviewed (approved):
Thanks, @jay-em-Ach 👍
